fdbserver/BlobManager.actor.cpp

```diff
@@ -1084,7 +1080,7 @@ ACTOR Future<Void> recoverBlobManager(BlobManagerData* bmData) {
 				std::tie(splitState, version) = decodeBlobGranuleSplitValue(split.value);
 				const KeyRange range = blobGranuleSplitKeyRangeFor(parentGranuleID);
 				if (splitState <= BlobGranuleSplitState::Initialized) {
-					bmData->workerAssignments.insert(range, UID());
+					workerAssignments.insert(range, UID());
```

Comment:
Add a comment here on why we're using the empty UID? That way you can understand this code without having to fully understand range assigner